src/pages/general/GuestInfo.tsx

```diff
@@ -0,0 +1,657 @@
+
+  return (
+    <List>
+      <ListItem>
+        <ListItemIcon>
+          <WristBand />
+        </ListItemIcon>
+        <ListItemText
+          primary={
+            <>
+              {guest && (
+                <span
+                  className={classes.termColorBadge}
+                  style={{
+                    background: wristBandPaletteColor(guest.term.guest_type)
+                      .main,
+                  }}
+                />
+              )}
+              {guestId}
+            </>
+          }
+          secondary="ゲスト ID"
+        />
+      </ListItem>
+      {guest && (
+        <>
+          <Grid container spacing={0}>
+            <Grid item xs={6}>
+              <ListItem>
+                <ListItemIcon>
+                  <Login />
+                </ListItemIcon>
+                <ListItemText
+                  primary={getStringDateTimeBrief(guest.entered_at)}
+                  secondary="入場時刻"
+                />
+              </ListItem>
+            </Grid>
+            <Grid item xs={6}>
+              <ListItem>
+                <ListItemIcon>
+                  <Logout />
+                </ListItemIcon>
+                <ListItemText
+                  primary={
+                    guest.exited_at
+                      ? getStringDateTimeBrief(guest.exited_at)
+                      : "-"
+                  }
+                  secondary="退場時刻"
+                />
+              </ListItem>
+            </Grid>
+          </Grid>
+          <Grid container spacing={0}>
+            <Grid item xs={6}>
+              <ListItem>
+                <ListItemIcon>
+                  <AccessTime />
+                </ListItemIcon>
+                <ListItemText
+                  primary={
+                    guest.term.enter_scheduled_time
+                      ? getStringDateTimeBrief(guest.term.enter_scheduled_time)
+                      : "-"
+                  }
+                  secondary="入場予定時刻"
+                />
+              </ListItem>
+            </Grid>
+            <Grid item xs={6}>
+              <ListItem>
+                <ListItemIcon>
+                  <AccessTime />
+                </ListItemIcon>
+                <ListItemText
+                  primary={
+                    guest.term.exit_scheduled_time
+                      ? getStringDateTimeBrief(guest.term.exit_scheduled_time)
+                      : "-"
+                  }
+                  secondary="退場予定時刻"
+                />
+              </ListItem>
+            </Grid>
+          </Grid>
+        </>
+      )}
+    </List>
+  );
+};
+
+const LogList: React.FC<{ logs: ActivityLog[]; status: AllStatus }> = ({
+  logs,
+  status,
+}) => (
+  <List>
+    {logs
+      .sort((a, b) => {
+        const timeA = moment(a.timestamp);
+        const timeB = moment(b.timestamp);
+        if (timeA.isSame(timeB)) return 0;
+        if (timeA.isBefore(timeB)) return -1;
+        return 1;
+      })
```

Comment:
```suggestion
      .sort((a, b) => moment(a.timestamp).diff(b.timestamp))
```
他のところで見逃してた気がしますが
https://gist.github.com/onildoaguiar/6cf7dbf9e0f0b8684eb5b0977b40c5ad